src/flamenco/runtime/program/fd_stake_program.c

```diff
@@ -2083,6 +2083,321 @@ redelegate( fd_exec_instr_ctx_t const * ctx,
   return 0;
 }
 
+// https://github.com/anza-xyz/agave/blob/cdff19c7807b006dd63429114fb1d9573bf74172/programs/stake/src/stake_state.rs#L136
+static int 
+move_stake_or_lamports_shared_checks( fd_exec_instr_ctx_t const *   invoke_context,
```

Comment:
its fine to call this instr_ctx

Reply:
we want to match the names used in agave (similar to other functions in the file)